.github/workflows/build_test.yml

```diff
@@ -133,7 +136,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [ 3.8, 3.9, "3.10", "3.11" ]
+        python-version: [3.9, "3.10", "3.11" ]
```

Comment:
We also need to change `index.rst` in the docs and possibly the readme regarding the Python versions we support.